solutions/usaco-921.mdx

```diff
@@ -76,9 +76,10 @@ vector<int> adj[MAXN];
 
 int timer = 1;
 // does initial DFS to find the time in/out for each subtree-range.
-void dfs(int v=0,int p=-1)
+void dfs(int v=0,int p=0)
 {
 	in[v] = timer++;
+	// prevent array out of bounds issues
```

Comment:
can you remove this comment? also, why were extra spaces added?

Reply:
where were the extra spaces? and yeah imma remove the comment